### PR TITLE
Add ability to open the Yarn homepage of a package

### PR DESCRIPTION
**Pull request by gregberge**
This package is very useful, thanks!

### CODE REVIEW

**Comment by gregberge** [comment]
Ping @sindresorhus

**Comment by sindresorhus** [comment]
Why? What benefits does the Yarn site have over npmjs.com? :)

**Comment by gregberge** [comment]
@sindresorhus a lof of useful informations

screen shot 2017-08-29 at 10 33 35

Personally I prefer it and I am not alone, see https://github.com/yarnpkg/yarn/issues/3696.

File: cli.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ const cli = meow(`
 
 	Options
 	  --github -g  Open the GitHub repo of the package
+		--yarn -y    Open the Yarn homepage of the package
```
Comment (sindresorhus) [inline cli.js:16]: Incorrect indentation here.

And you need to define `-y` in the below `alias` definition.

**Comment by sindresorhus** [comment]
Alright :)

You also need to update the readme description, package.json description, and the CLI help output in the readme. And remove the `yarn.lock` file (never commit that to projects if they don't already have one).

**Comment by gregberge** [comment]
@sindresorhus all done!

**Comment by sindresorhus** [comment]
Thanks :)

**Comment by gregberge** [comment]
@sindresorhus you should also update GitHub description.

**Comment by sindresorhus** [comment]
Already done

**Comment by gregberge** [comment]
@sindresorhus small bug https://github.com/sindresorhus/npm-home/pull/4

**Pull request merged**